### DISCUSSION TITLE
Welcome to pg_timetable Discussions!

### DISCUSSION THREAD

**Discussion by pashagolub** [cybertec-postgresql/pg_timetable#169] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MichaelDBA** [comment]
SO what is the link to get to this discussion area?

**Reply by pashagolub** [reply]
You're already here :-)

**Reply by pashagolub** [reply]
https://github.com/cybertec-postgresql/pg_timetable/discussions

**Comment by adambir** [comment]
Hello, I would like to know when v4 release of pg_timetable is scheduled for ?

**Reply by pashagolub** [reply]
Hello @adambir,

We hope it should be a mid-July. The functionality planned is implemented, but we want to give it some more tests due to major changes.
